srcpkgs/dracut/files/kernel-uefi-hook-postinst

```diff
@@ -0,0 +1,27 @@
+#!/bin/sh
+#
+# Kernel post-install hook for dracut EFI bundle.
+#
+# Arguments passed to this script: $1 pkgname, $2 version.
+#
+PKGNAME="$1"
+VERSION="$2"
+
+. "${ROOTDIR}/etc/default/dracut-uefi-hook"
+if [ "x${CREATE_UEFI_BUNDLES}" != x1 ]; then
+	exit 0
+fi
+
+if [ ! -x bin/dracut ]; then
+	exit 0
+	fi
+
+if [ -z "${KERNEL_CMDLINE}" ] && [ "x${SKIP_KERNEL_CMDLINE}" != x1 ]; then
+	echo "ERROR: You need to set the kernel command line in /etc/default/dracut-uefi-hook"
```

Comment:
Is this logic necessary?

Reply:
It's solving the issue that if dracut doesn't find a `kernel_cmdline` entry in a `dracut.conf` file and isn't passed a cmdline in the command line argumets, it will prompt the user for a cmdline.

Reply:
I couldn't think of another way of solving this particular issue.

Reply:
The other way of solving this would be always passing at least an empty string to the command line parameter.

Reply:
That seems like a good option, provided it has no unanticipated side effects.

Reply:
I would still prefer erroring out, but if you think it's fine.

Reply:
Hmm there's also the use case without secure boot that allows the bundle to use the cmdline from refind/grub, so the command line could be empty.

Reply:
Just write a warning in the configuration file, and let the user see the boot failure :shrug: 
Don't bother with it, I would say.